HHS/Head-Start-TTADP
# [TTAHUB-513] add percentage of recipients served to dashboard

## Description of change

Add new tile to dashboard overview widget displaying the percentage of recipients served an activity report out of total unique recipients. 

Add new filters to the grant scopes so that certain AR filters would also change this number.

## How to test
 View the dashboard overview and check the numbers. Add applicable filters and verify any changed numbers.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-513

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
